flatten_array.rb

```diff
@@ -0,0 +1,28 @@
+require 'pry'
+
+class FlattenArray
+  def self.flatten(array)
+    flattened_array = []
+
+    array.each do |element|
+      flattened_array = self.add_element_array(element, flattened_array)
+    end
```

Comment:
You might want to add a new line here.
```suggestion
    end
    
```